src/executables/unload_schema.c

```diff
@@ -1148,6 +1150,15 @@ emit_schema (print_output & output_ctx, DB_OBJLIST * classes, int do_auth, DB_OB
 	    {
 	      output_ctx (", COLLATE %s", lang_get_collation_name (class_->collation_id));
 	    }
+
+	  if (class_ != NULL)
+	    {
+	      tde_algo_name = tde_get_algorithm_name ((TDE_ALGORITHM) class_->tde_algorithm);
+	      if (strcmp (tde_algo_name, tde_get_algorithm_name (TDE_ALGORITHM_NONE)) != 0)
```

Comment:
tde_get_algorithm_name () may return NULL. the behavior of strcmp is undefined in case of NULL is passed to an argument,
so depending on the implementation, strcmp may cause a segmentation fault.

Reply:
It is impossible to get NULL while getting the name of the algorithm from `tde_algorithm` of a class object.
I added some asserts for that instead. Refer to https://github.com/CUBRID/cubrid/pull/2450/commits/42366e49c402bad3d211957b5158ce227b439193. Thanks.